src/views/TeamStatisticsPanel/teamstatistics.json

```diff
@@ -0,0 +1,22 @@
+[
```

Comment:
Let's use a JS module instead of a JSON file for this data. Take a look at the [`SchoolsData.js`](https://github.com/the-coding-school/the-cs/blob/master/src/views/SchoolsPanel/SchoolsData.js) file for reference.

It'll allow us to avoid the following line in `TeamStatisticsPanel.js`:
```js
const statsIconRoot = process.env.PUBLIC_URL + '/images/statistics/';
```